app/uk/gov/hmrc/agentfirelationship/controllers/RelationshipController.scala

```diff
@@ -47,16 +47,12 @@ class RelationshipController @Inject()(gg: GovernmentGatewayProxyConnector,
   }
 
   def createRelationship(arn: String, service: String, clientId: String): Action[AnyContent] = Action.async { implicit request =>
-    val maximumRelationshipCount = 2
     val relationship: Relationship = Relationship(Arn(arn), service, clientId)
 
     (for {
       relationshipList <- mongoService.findAllRelationshipsForAgent(arn)
       existingRelationship <- mongoService.findRelationships(relationship)
     } yield (relationshipList.length, existingRelationship.nonEmpty)) flatMap {
```

Comment:
The length of the relationship list is only being retrieved to check that the agent is not going above their maximum relationship count. If the maximum is being removed, we should remove this call and re-write this section